api-gateway/src/main/resources/static/scripts/bills-by-owner-id/bills-by-owner-id.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+angular.module('billsByOwnerId', ['ui.router'])
+    .config(['$stateProvider', function ($stateProvider) {
+        $stateProvider
+            .state('billsByOwnerId', {
+                parent: 'app',
+                url: '/bills/customerId/:customerId',
```

Comment:
[Nitpick] inconsistency between the name of the state and the URL changing from ownerId to customerId

Reply:
naming is inconsistent in customer service so I made the links follow their naming